src/pages/dar_application/collaborator/ApproverStatus.tsx

```diff
@@ -35,16 +51,16 @@ export default function ApproverStatus(props: ApproverStatusProps): React.JSX.El
                 the PI designates to download data and/or share the requested data with other Internal Lab Staff
                 (ie., staff members and trainees under the direct supervision of the PI).`}
                 options={[
-                    {name: 'yes', text: 'Yes'},
-                    {name: 'no', text: 'No'}
+                    {name: 'true', text: 'Yes'},
+                    {name: 'false', text: 'No'}
```

Comment:
Can we instead make these `true` or `false` booleans, instead of strings?

Reply:
Unfortunately no.  The underlying form control seems like it's dependent on strings.  It starts blowing up when I set these as booleans.